src/modules/roc_fec/writer.cpp

```diff
@@ -37,12 +37,17 @@ Writer::Writer(const WriterConfig& config,
     , buffer_pool_(buffer_pool)
     , repair_block_(allocator)
     , first_packet_(true)
-    , cur_sbn_((packet::blknum_t)core::random(0, packet::blknum_t(-1)))
-    , cur_block_repair_sn_((packet::seqnum_t)core::random(0, packet::seqnum_t(-1)))
     , cur_packet_(0)
     , fec_scheme_(fec_scheme)
     , valid_(false)
     , alive_(true) {
+    uint32_t rand_cur_sbn_ = 0, rand_cur_block_repair_sn_ = 0;
```

Comment:
We use tailing underscore only for private class fields and methods. Local variables should not have it. See here: https://roc-project.github.io/roc/docs/development/coding_guidelines.html

So `rand_cur_sbn_` -> `rand_cur_sbn` and `rand_cur_block_repair_sn_` -> `rand_cur_block_repair_sn`.